integration/nwo/deploy.go

```diff
@@ -84,6 +84,31 @@ func DeployChaincode(n *Network, channel string, orderer *Orderer, chaincode Cha
 	}
 }
 
+// DeployChaincodeWithoutInitialization is a helper that will install chaincode to all peers that
+// are connected to the specified channel, approve the chaincode on one of the
+// peers of each organization in the network, commit the chaincode definition
+// on the channel using one of the peers, and wait for the chaincode commit to
+// complete on all of the peers.
+// This method does not invoke initialization of chaincode. This is useful for chaincode as extenal service
+// where package code is required for stating the external chaincode service.
```

Comment:
This condition is for exactly one scenario. It doesn't warrant duplicating `DeployChaincode` in the same package that hods it.